demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<p>The StellarGraph links here refer to a specific PR here when I test; I think they should point to master? They don't seem to work correctly inside the github notebook renderer either, eg: <a href="https://github.com/stellargraph/stellargraph/blob/cf18446ab0c9f5ba553c1c38c2cf6e62537fc655/demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb" target="_blank">https://github.com/stellargraph/stellargraph/blob/cf18446ab0c9f5ba553c1c38c2cf6e62537fc655/demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb</a></p><p> </p><p>Also the display of "This demo..." and "StellarGraph supports..." paragraphs looks as though they're quoted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
> The StellarGraph links here refer to a specific PR here when I test; I think they should point to master? They don't seem to work correctly inside the github notebook renderer either, eg: https://github.com/stellargraph/stellargraph/blob/cf18446ab0c9f5ba553c1c38c2cf6e62537fc655/demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

Could be more specific about which links and their destination that you find unexpected?

I clicked on every link in this section in the GitHub rendering you linked, and all but one went where I expected:

| name | status |
|---|---|
| See all other demos | ✅ |
| The StellarGraph library | ✅  |
| graphs | ❌[1] |
| https://linqs.soe.ucsc.edu/data | ✅ |
| "Knowing Your Neighbours: Machine Learning on Graphs" | ✅ |
| Pandas | ✅ |
| scikit-learn | ✅ |
| StellarGraph | ✅ |
| TensorFlow Keras | ✅ |
| other algorithms for node classification |  ✅ |
| other tasks | ✅ |
| Graph Convolutional Networks (GCN): Semi-Supervised Classification with Graph Convolutional Networks | ✅ |

1. pointed to <https://en.wikipedia.org/wiki/Graph_%5C(discrete_mathematics%5C>, fixed by updating to <https://en.wikipedia.org/wiki/Graph_%28discrete_mathematics%29>)


> Also the display of "This demo..." and "StellarGraph supports..." paragraphs looks as though they're quoted?

The quoting is on purpose: they're meant to be asides and/or providing quick context/helpful links, but there's no markdown to generate the real semantic markup (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside) and writing the HTML directly doesn't for what I want with this (a "side note" providing more info):

- there's no special rendering for `<aside>` in jupyter lab: ![image](https://user-images.githubusercontent.com/1203825/77375684-524bbe00-6dc2-11ea-9b10-91a8008063af.png)
- GitHub strips out the `<aside>` tag: https://gist.github.com/huonw/87106c1524cd99e73f8db3c53afc1d26

Reply:
Thanks for explaining about the quotes. The bad links were in the cell I think NBreview referenced, specifically the paragraph:

```StellarGraph supports other algorithms for doing node classification, as well as many other tasks such as link prediction, community detection, and graph classification.```

Reply:
Ah, right. That's https://github.com/stellargraph/stellargraph/issues/1139. I'll remove it.